packages/wp-dashboard/src/components/editorSettings/customFonts/stories/index.js

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import { useCallback, useRef, useState } from '@web-stories-wp/react';
+import { action } from '@storybook/addon-actions';
+
+/**
+ * Internal dependencies
+ */
+import { rawCustomFonts } from '../../dataUtils/formattedCustomFonts';
+import CustomFontsSettings from '..';
+
+export default {
+  title: 'Dashboard/Views/EditorSettings/CustomFonts',
+  component: CustomFontsSettings,
+};
+
+export const _default = () => {
+  const [addedFonts, setAddedFonts] = useState(rawCustomFonts);
```

Comment:
@BrittanyIRL I wanted to add the new settings area to the storybook but I'm not actually certain where in the Storybook it's displayed at all. Is this currently not visible in the Storybook due to refactoring everything WP related out of the `dashboard` package to `wp-dashboard`, or am I just not able to find it?

Reply:
Looks like an oversight from decoupling, any stories within `wp-dashboard` are missing.

To fix, simply add `'../packages/wp-dashboard/src/**/stories/*.@(js|mdx)'` here:

https://github.com/google/web-stories-wp/blob/08c0fa29154e2667dbf71a3651671ab4bc43f232/.storybook/main.cjs#L18-L25

cc @sayedtaqui FYI